opennms-services/src/main/java/org/opennms/netmgt/poller/Poller.java

```diff
@@ -426,7 +426,7 @@ private void scheduleExistingServices() throws Exception {
      * @param nodeLocation a {@link String} object.
      * @param ipAddr a {@link String} object.
      * @param svcName a {@link String} object.
-     * @param pollableNode
+     * @param pollableNode a {@link PollableNode} object
```

Comment:
Not quite what I had in mind :)

Reply:
Of course not, I have added this comment. I can extend it bit more in detail. https://github.com/OpenNMS/opennms/pull/2988/commits/dc3d858cde36390b5e77ada1a0f40f1031619756#diff-ce272af2d0a2ea89b38de1d671441e79R443

Reply:
Added more detailed comment.